fluent-plugin-kafka.yaml

```diff
@@ -0,0 +1,64 @@
+package:
+  name: fluent-plugin-kafka
+  version: 0.19.3
+  epoch: 0
+  description: Fluentd plugin for Apache Kafka that forwards logs to Kafka.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    provides:
+      - ruby3.2-fluent-plugin-kafka=${{package.version}}-r${{package.epoch}}
+    runtime:
+      - ruby3.2-fluentd
+      - ruby3.2-kafka
+  target-architecture:
+    - aarch64
+    - x86_64
+
+environment:
+  contents:
+    repositories:
+      - https://packages.wolfi.dev/os  # Ensure this URL is correct
+    keyring:
+      - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - git
+      - ruby-3.2
+      - ruby-3.2-dev
+
+
+vars:
+  gem: fluent-plugin-kafka
+
+pipeline:
+  - uses: git-checkout
+    with:
+      destination: ${{vars.gem}}
+      repository: https://github.com/fluent/fluent-plugin-kafka
+      tag: v${{package.version}}  # Automatically checks out the tag corresponding to the version
+
+  - working-directory: ${{vars.gem}}
+    pipeline:
+      - runs: |
+          latest_commit=$(git rev-list -n 1 --tags --skip=0)
+          echo "Latest commit for the tag: $latest_commit"
+      - uses: ruby/build
+        with:
+          gem: ${{package.name}}
+      - uses: ruby/install
+        with:
+          gem: ${{package.name}}
+          version: ${{package.version}}
+      - uses: ruby/clean
+
+update:
+  enabled: true
+  github:
+    identifier: fluent/fluent-plugin-kafka
+    strip-prefix: v
+    use-tag: true
+
+
```

Comment:
remove this extra space otherwise yam linting will complain. We lint using https://github.com/chainguard-dev/yam so linting can be done using `yam foo.yaml` 

if you don't have it installed use the sdk image using `make dev-container` you'll have it installed there `make dev-container` and `yam foo.yaml`